README.md

```diff
@@ -14,8 +28,86 @@ depend on the number of vertices, the number of edges, or both?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+### Answer
+
+My Implementation for this algorithm only depends upon the number of vertices and not at all on the number of edges. This is because an Adjacency matrix is built up of rows and columns from the vertices of the graph.And though the number in each entry depicts a relationship for a Edge exsisting or not the size of the array is not affected by these.
+
+
+I used three for loops in my code. They first two loop over the rows and columns of the Adjacecency Matrix to convert enties into an Adjacency List. Additionally , I also have some conditionals if-statements inside these loops. One of the conditionals uses a function called NoEdges and loops through the given row at the $ x'th $ iteration and checks if that element is all zeros, if so we append an empty list. This gives a time complexity of $V \cdot V$ For the inner loop and the conditional.
```

Comment:
I don't believe that's true anymore? The code you give below is different to the actual code.